spyder/utils/programs.py

```diff
@@ -772,44 +755,30 @@ def run_python_script_in_terminal(fname, wdir, args, interact,
                                    "an external terminal"),
                                  QMessageBox.Ok)
     elif sys.platform.startswith('linux'):
-        programs = [{'cmd': 'gnome-terminal',
-                     'wdir-option': '--working-directory',
-                     'execute-option': '-x'},
-                    {'cmd': 'konsole',
-                     'wdir-option': '--workdir',
-                     'execute-option': '-e'},
-                    {'cmd': 'xfce4-terminal',
-                     'wdir-option': '--working-directory',
-                     'execute-option': '-x'},
-                    {'cmd': 'xterm',
-                     'wdir-option': None,
-                     'execute-option': '-e'},]
+        programs = [{'cmd': 'gnome-terminal', 'execute-option': '-x'},
+                    {'cmd': 'konsole', 'execute-option': '-e'},
+                    {'cmd': 'xfce4-terminal', 'execute-option': '-x'},
+                    {'cmd': 'xterm', 'execute-option': '-e'}]

```

Comment:
Why did you remove the `wdir-option` from these dicts?

Reply:
The working directory shell command option is redundant with the `cwd` keyword option to Popen. I chose to use `cwd` instead of the shell command option to simplify the code. I was able to verify behavior on my Ubuntu VM.

Reply:
Is there some other issue that I missed?

Reply:
No, I just wanted to check with you what you confirmed above, i.e.

> The working directory shell command option is redundant with the cwd keyword option to Popen